src/abstract-list-field/abstract-list-field.component.ts

```diff
@@ -75,12 +76,20 @@ export abstract class AbstractListFieldComponent extends AbstractFieldComponent
   /**
    * Returns path of the property of an element at index.
    */
-  getValuePath(index: number, property: string): Array<any> {
-    return this.path.concat(index, property);
+  getValuePath(index: number, property?: string): Array<any> {
```

Comment:
Actually can you override this without `property` param for `primitive-list` because looks like we're gonna have 2 conditions and it's better if we separate. In that case you don't have to use `||` in `${property || ''}` that I suggested in another comment below.